packages/catana/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=catana
+pkgver=1.0
+pkgrel=1
+pkgdesc='CATANA filters your wordlist according to the specified password policy.'
+groups=('blackarch' 'blackarch-misc')
+arch=('any')
+url='https://github.com/D3vil0per/catana'
+license=('GNU General Public License v3.0')
+depends=('bash')
+makedepends=('git')
+source=("git+https://github.com/D3vil0per/$pkgname.git")
+
+pkgver() {
+	cd $pkgname
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+	cd $pkgname
+
+  install -Dm 755 -t "$pkgdir/usr/bin/" catana
+}
+sha256sums=('SKIP')
```

Comment:
could you put it under source please and we use `sha512sums`

Reply:
Commit 4eb29ca65934a806b201d6c3e6e452b3b8fc0640